js/foundation.tooltip.js

```diff
@@ -135,6 +135,14 @@ class Tooltip extends Positionable {
     }
 
     var _this = this;
+
+    // Update Tooltip text if title attribute has changed since init 
+    if (this.$element.attr('title') !== '') {
+      this.options.tipText = this.$element.attr('title');
+      this.template.text(this.options.tipText);
+      this.$element.attr('title', '');
+    }
```

Comment:
This seems a bit duplicated with the initialization. Also, this does not support HTML.
I think we could factorize this and

https://github.com/zurb/foundation-sites/blob/53f0ee83aa0963f0dde56811277f03cfaa29c130/js/foundation.tooltip.js#L52-L60

.... as a `_reflowTitle` function, and maybe add a `_reflow` also.

Reply:
@ncoden I agree. If `allowHTML` is true, can the HTML be set through the title attribute or does that only work by setting it through the `tipText` option?

Reply:
For a simple API, I would say that `title` and `tipText` should be considered as aliases. But using `title` for html content is not a good practice and we may not allow that....